MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -552,120 +551,116 @@ private void listItems(int id) {
           new Intent(MainActivity.this, MapillaryActivity.class),
-          R.string.activity_lab_mapillary_url
-        ));
+          R.string.activity_lab_mapillary_url, false, BuildConfig.MIN_SDK_VERSION));
+
         exampleItemModel.add(new ExampleItemModel(
           R.string.activity_lab_los_angeles_tourism_title,
           R.string.activity_lab_los_angeles_tourism_description,
           new Intent(MainActivity.this, LosAngelesTourismActivity.class),
-          R.string.activity_lab_los_angeles_tourism_url
-        ));
+          R.string.activity_lab_los_angeles_tourism_url, false, BuildConfig.MIN_SDK_VERSION));
+
         exampleItemModel.add(new ExampleItemModel(
           R.string.activity_lab_indoor_map_title,
           R.string.activity_lab_indoor_map_description,
           new Intent(MainActivity.this, IndoorMapActivity.class),
-          R.string.activity_lab_indoor_map_url
-        ));
+          R.string.activity_lab_indoor_map_url, false, BuildConfig.MIN_SDK_VERSION));
+
         exampleItemModel.add(new ExampleItemModel(
           R.string.activity_lab_location_picker_title,
           R.string.activity_lab_location_picker_description,
           new Intent(MainActivity.this, LocationPickerActivity.class),
-          R.string.activity_lab_location_picker_url
-        ));
+          R.string.activity_lab_location_picker_url, false, BuildConfig.MIN_SDK_VERSION));
+
         exampleItemModel.add(new ExampleItemModel(
           R.string.activity_lab_marker_following_route_title,
           R.string.activity_lab_marker_following_route_description,
           new Intent(MainActivity.this, MarkerFollowingRouteActivity.class),
-          R.string.activity_lab_marker_following_route_url
-        ));
+          R.string.activity_lab_marker_following_route_url, false, BuildConfig.MIN_SDK_VERSION));
+
         exampleItemModel.add(new ExampleItemModel(
           R.string.activity_lab_space_station_location_title,
           R.string.activity_lab_space_station_location_description,
           new Intent(MainActivity.this, SpaceStationLocationActivity.class),
-          R.string.activity_lab_space_station_location_url
-        ));
-        if (Build.VERSION.SDK_INT >= 26) {
-          exampleItemModel.add(new ExampleItemModel(
-            R.string.activity_lab_picture_in_picture_title,
-            R.string.activity_lab_picture_in_picture_description,
-            new Intent(MainActivity.this, PictureInPictureActivity.class),
-            R.string.activity_lab_picture_in_picture_url, true
-          ));
-        }
+          R.string.activity_lab_space_station_location_url, false, BuildConfig.MIN_SDK_VERSION));
+
+        exampleItemModel.add(new ExampleItemModel(
+          R.string.activity_lab_picture_in_picture_title,
+          R.string.activity_lab_picture_in_picture_description,
+          new Intent(MainActivity.this, PictureInPictureActivity.class),
+          R.string.activity_lab_picture_in_picture_url, false, 26));
```

Comment:
Instead of hardcoding the number, could you use `Build.VERSION_CODES` instead?

Reply:
Ok, the hardcoded 26 is now `Build.VERSION_CODES.O`

I had to upgrade the app's `supportLibVersion`, `compileSdkVersion`, and `buildToolsVersion` version #s to get access to the O-level version code 👇 

<img width="301" alt="screen shot 2017-10-19 at 3 30 33 pm" src="https://user-images.githubusercontent.com/4394910/31797315-7b2c05be-b4e2-11e7-9f6e-8064ae095d46.png">